.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Eslint config. Loading airbnb as the default since that is a popular one, and then overriding from there.